client/src/components/Translations.jsx

```diff
@@ -17,13 +17,20 @@ const useStyles = makeStyles(theme => ({
 export default function TranslationsMenu() {
   const classes = useStyles();
   const { i18n, t } = useTranslation();
+  const [loaded, setLoaded] = useState(false);
+
+  const setLanguage = () => {
+    setLoaded(true)
+    i18n.changeLanguage(window.navigator.language.slice(0, 2));
```

Comment:
Is every language / country code 2 digits?

Reply:
![](https://i.ibb.co/ykdSds0/Screen-Shot-2020-05-05-at-8-53-19-PM.png)
I add Arabic, Chinese and Russian to my browser and it show that language codes consistent.  It's or `en` or `en-US` format.

Reply:
I'd say that's good for now then 👍